config/environments/test.rb

```diff
@@ -62,11 +62,14 @@
   # Feature Toggle for UMM-S
   config.umm_s_enabled = true
 
+  # Feature Toggle for UMM-Var Generation
+  config.uvg_enabled = true
+
   config.cmr_env = 'sit'
   config.echo_env = 'sit'
   config.urs_register_url = 'https://sit.urs.earthdata.nasa.gov/users/new'
 
   config.middleware.use RackSessionAccess::Middleware
 
-  config.colorize_logging = false
+  # config.colorize_logging = false
```

Comment:
a non-blocking "why" (will approve but I just want to know what's up here)

Reply:
So I hadn't actually checked our test logs until fixing all the headless chrome tests. Those logs aren't colored like our dev logs and I hadn't realized this is where that was controlled, so I thought it would be nice to see if having them be colored would be helpful. I would like to keep this commented for a little while to determine that.